rplidar.go

```diff
@@ -51,6 +51,8 @@ var (
 	Model = resource.NewModel("viam", "lidar", "rplidar")
 	// rplidarModelByteMap maps the byte model representation to a string representation
 	rplidarModelByteMap = map[byte]string{24: "A1", 49: "A3", 97: "S1"}
+	// The max capture frequency for rplidar models based on datasheets
+	maxScanningFrequencyByModel = map[string]float64{"A1": 10, "A3": 15, "S1": 15}
```

Comment:
[opt/nit] Since we're now using the models in multiple places, I'm in favor of making them constants